Description: Fix build failure with GCC 4.8
   This patches fixes the following build failure:
   
   ../../dix/window.c:887:5: error: implicit declaration of function 'DeleteWindowFromAnySelections' [-Werror=implicit-function-declaration]
   
   by including "selection.h", where the function is prototyped.

Author: Gerardo Malazdrewicz <gerardo@malazdrewicz.com.ar>
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=701372
--- xorg-server-1.12.4/dix/window.c	2012-05-17 14:09:02.000000000 -0300
+++ xorg-server-1.12.4.good/dix/window.c	2013-04-18 11:53:06.787521781 -0300
@@ -131,6 +131,8 @@
 #include "xace.h"
 #include "exevents.h"
 
+#include "selection.h"
+
 #include <X11/Xatom.h>          /* must come after server includes */
 
 /******
